.gitignore

```diff
@@ -1,29 +1,19 @@
 # git ignore patterns
 
-autom4te.cache/
 libltdl/
 
-Makefile.in
-
-aclocal.m4
-compile
-config.guess
-config.sub
-configure
-depcomp
-install-sh
-ltmain.sh
-missing
-
 pynest/pynestkernel.cpp
 
 *.o
 *.lo
 *.la
 .*
 *.log
+<<<<<<< HEAD
 *.pyc
 config.status
+=======
+>>>>>>> master
 Makefile
```

Comment:
These are more artefacts from an unresolved merge conflict. Please remove them and also the line with `config.status`.